packages/client/tests/functional/typescript/tests.ts

```diff
@@ -35,9 +35,12 @@ function getAllTestSuiteTypeChecks(fileNames: string[]) {
 }
 
 describe('typescript', () => {
-  const suitePaths = glob.sync('./**/.generated/**/*.ts', {
-    ignore: ['./**/.generated/**/*.d.ts', './**/.generated/**/_schema.ts'],
-  })
+  const suitePaths = glob
+    .sync('./**/.generated/**/*.ts', {
+      ignore: ['./**/.generated/**/*.d.ts', './**/.generated/**/_schema.ts'],
+    })
+    .concat([path.resolve(__dirname, '..', 'globals.d.ts')])
```

Comment:
Why does it need type checking?

Reply:
It does not really need typechecking, but since nobody really directly imports it it has no be included in the source set so global definitions for `testIf` and `describeIf` will be discovered.

Reply:
Makes sense. Can you please add a short comment above concat about this?